bin/phase-helpers.sh

```diff
@@ -655,8 +655,8 @@ econf() {
 			fi
 
 			if ___eapi_econf_passes_--disable-static; then
-				if [[ ${conf_help} == *--disable-static* || \
-						${conf_help} == *--enable-static* ]]; then
+				if [[ ${conf_help} == *--enable-shared\[=PKGS\]* &&
+						${conf_help} == *--enable-static\[=PKGS\]* ]]; then
```

Comment:
If we are to follow what has been proposed for the spec, then it should be like this:
```suggestion
				if [[ ${conf_help} == *--enable-shared[^A-Za-z0-9+_.-]* &&
						${conf_help} == *--enable-static[^A-Za-z0-9+_.-]* ]]; then
```